project/src/ui/popup/popup_window.gd

```diff
@@ -77,16 +49,16 @@ func _buttonOK_pressed() -> void:
 	
 func _buttonYes_pressed() -> void:
 	choice = true
-	emit_signal("click",choice)
+	emit_signal("click")
 	queue_free()
 	
 func _buttonNo_pressed() -> void:
 	choice = false
-	emit_signal("click",choice)
+	emit_signal("click")
 	queue_free()
 	
 # Returns user's choice (only for confirmation popup)
-func choiseRet():
+func choice_Ret():
```

Comment:
Change the calls to the function aswell, otherwise the program will brake. I have used it in fileTree for instance. 
And should it not be `choice_ret`?